### ISSUE TITLE
broken in latest version

### ISSUE THREAD

**Issue by backwardspy** [catppuccin/github#34]
attempting to apply the style as of 35e41f33 (#32) results in the following message:

```
Stylus failed to parse usercss:

stylus:50:10
   46|         $mantle = #e6e9ef
   47|         $crust = #dce0e8
   48|         $shadow = #dbdfef
   49|     }
   50|     else if (theme=="Frappe") {
----------------^
   51|         $type = dark
   52|         
   53|         $rosewater = #f2d5cf

unexpected "else"
```

this renders the style uninstallable (as far as i can tell!)

applying the style as it was at 011e121d works as expected.

**Comment by andreasgrafen** [comment]
Great catch, seems that somehow the formatting screwed up with stylus. This should be fixed with [284add2](https://github.com/catppuccin/github/commit/284add2651aa97fd45ec4f302086e48ac9d8caeb). c:

**Comment by backwardspy** [comment]
looks good to me, thanks!
